src/app/static/src/tests/components/header/fileMenu.test.ts

```diff
@@ -111,79 +111,79 @@ describe("File menu", () => {
-    //             }
-    //         });
-    //     await flushPromises();
-    //     await wrapper.find(".dropdown-toggle").trigger("click");
-    //     expect(wrapper.find(".dropdown-menu").classes()).toStrictEqual(["dropdown-menu", "show"]);
-    //     let link = wrapper.findAll(".dropdown-item")[1];
-    //     await link.trigger("mousedown");
-    //     await expectTranslated(link, "SaveJSON", "SauvegarderJSON", "GuardarJSON", store as any);
-
-    //     const hiddenLink = wrapper.find({ref: "save"});
-    //     expect(hiddenLink.attributes("href")).toBe("http://localhost#1234");
-
-    //     const re = new RegExp("naomi-(.*)\.json");
-    //     expect((hiddenLink.attributes("download") as string).match(re)).toBeDefined();
-
-    //     const expectedJson = {
-    //         state: {
-    //             baseline: {selectedDataset: null, selectedRelease: null, selectedDatasetIsRefreshed: false},
-    //             modelRun: mockModelRunState(),
-    //             modelCalibrate: {result: null, calibratePlotResult: null, comparisonPlotResult: null},
-    //             metadata: mockMetadataState(),
-    //             surveyAndProgram: {selectedDataType: null, warnings: []},
-    //             language: Language.en
-    //         },
-    //         files: {
-    //             pjnz: {hash: "2csv", filename: "2.csv"},
-    //             population: {hash: "1csv", filename: "1.csv"},
-    //             shape: {hash: "3csv", filename: "3.csv"},
-    //             survey: {hash: "4csv", filename: "4.csv"},
-    //             programme: {hash: "5csv", filename: "5.csv"},
-    //             anc: {hash: "6csv", filename: "6.csv"}
-    //         }
-    //     };
-
-
-    //     const actualBlob = (mockCreateObjectUrl as jest.Mock).mock.calls[0][0];
-    //     const reader = new FileReader();
-
-    //     reader.addEventListener('loadend', function () {
-    //         const text = reader.result as string;
-    //         const result = JSON.parse(text)[1];
-    //         expect(JSON.parse(result)).toStrictEqual(expectedJson);
-    //     });
-
-    //     await readAsText(reader, actualBlob);
-    // });
-
-    // it("aria-label and link text are translated for Json load", async () => {
-    //     switches.loadJson = true
-    //     const store = createStore();
-    //     const wrapper = mountWithTranslate(FileMenu, store, {
-    //         global: {
-    //             plugins: [store]
-    //         }, 
-    //     });
-    //     const link = wrapper.findAll(".dropdown-item")[2];
-    //     await expectTranslated(link, "LoadJSON", "ChargerJSON", "CarregarJSON", store as any);
-    //     const input = wrapper.find("#upload-file")
-    //     await expectTranslated(input,
-    //         "Select file",
-    //         "Sélectionner un fichier",
-    //         "Selecionar ficheiro",
-    //         store as any,
-    //         "aria-label");
-    // });
+    it("downloads JSON file", async () => {
```

Comment:
Wow, when did all this get commented out!? Was it a casualty of the Great Vue3 Campaign?

Reply:
Yeah I think so! Will be nice to cut it all out after this PR :dagger: